namer/types.py

```diff
@@ -16,8 +16,10 @@
 from typing import List, Optional, Sequence
 
 from loguru import logger
-from pathvalidate import Platform, sanitize_filename
-from requests_cache import BACKEND_CLASSES, BaseCache, CachedSession
```

Comment:
old is better

Reply:
yeah, it's an error in pylance though, not sure how to ignore the error

Reply:
looks like pylance lying to you, `pathvalidate` have this imports in `__init__`, https://github.com/thombashi/pathvalidate/blob/master/pathvalidate/__init__.py